corehq/motech/repeaters/models.py

```diff
@@ -65,6 +65,7 @@
 """
 import inspect
 import json
+import traceback
```

Comment:
**Today I Learned** that if a test passes locally but fails on GitHub, it could be because, when GitHub merges in master before it runs the tests, that merge deletes an import that you need!